### PR TITLE
Aden-13707 Postpone gallery slots bidding

### PR DESCRIPTION
**Pull request by lnawrotek**

### CODE REVIEW

File: src/ad-bidders/bidder-helper.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		}
+	}
+
+	return slotName;
+}
+
+export function hasCorrectBidGroup(code: string, group: string): boolean {
+	const slotBidGroup = getSlotBidGroup(code);
+
+	return slotBidGroup && slotBidGroup === group;
```
[the remark this answers is not in the document]
Comment (lnawrotek) [reply src/ad-bidders/bidder-helper.ts]: I had to change this method, it has new logic now

File: spec/ad-bidders/bidder-helper.spec.ts [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	getSlotNameByBidderAlias,
+	hasCorrectBidGroup,
+} from '@wikia/ad-bidders/bidder-helper';
+import { context } from '@wikia/core';
+import { expect } from 'chai';
+
+describe('bidder-helper', () => {
+	describe('getSlotNameByBidderAlias', () => {
+		it('should return slot name if has no alias', () => {
+			global.sandbox.stub(context, 'get').withArgs('slots.slot1').returns({ uid: '12345' });
```
[the remark this answers is not in the document]
Comment (lnawrotek) [reply spec/ad-bidders/bidder-helper.spec.ts:13]: It's not necessary, Mocha Hook restores sandbox (`spec/helpers/sandbox-hook.js`)

File: src/ad-bidders/bidder-helper.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import { context, slotService } from '@ad-engine/core';
+
+export const defaultSlotBidGroup = 'not-defined';
+
+export function getSlotNameByBidderAlias(id: string): string {
```
[the remark this answers is not in the document]
Comment (lnawrotek) [reply src/ad-bidders/bidder-helper.ts]: It's not my code, I've moved it, but refactoring is always a good idea. Please check it again and compare these two functions.

File: src/ad-bidders/bidder-helper.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import { context, slotService } from '@ad-engine/core';
```
Comment (lnawrotek) [inline src/ad-bidders/bidder-helper.ts:1]: I've moved some methods from `src/ad-bidders/alias-helper.ts` to this file

**Review by preludek — APPROVED** [review]

**Pull request merged**